src/advanced_inv.cpp

```diff
@@ -1985,7 +1985,8 @@ bool advanced_inventory::move_content( item &src_container, item &dest_container
     }
 
     std::string err;
-    if( !dest_container.fill_with( src, err ) ) {
+    // @todo Allow buckets here, but require them to be on the ground or wielded
+    if( !dest_container.fill_with( src, err, false ) ) {
```

Comment:
This parameter being a boolean is the only thing I don't like about the implementation. I just can't tell the difference between `fill_with( src, err, true )` and `fill_with( src, err, false )`. At least not without looking up the function definition.